src/wordFormatter.js

```diff
@@ -0,0 +1,19 @@
+export class WordFormatter {
+
+  constructor(guessRegister) {
+    this.guessRegister = guessRegister
+  }
+
+  formattedWord(secretWord) {
+    let secretWordLetters = secretWord.split("");
```

Comment:
could this be a `const` rather than a `let`? I think it's best to use as many `const` as possible because that means it minimises any mutable state - and mutable state can often cause bugs...